.github/workflows/lighthouse.yml

```
@@ -0,0 +1,104 @@
+        with:
+          configPath: ./.lighthouserc.json
+          temporaryPublicStorage: true
+          uploadArtifacts: true
+        env:
+          # startServerCommand(pnpm start)가 런타임에 참조하므로 동일한 더미 값을 준다.
+          NEXT_PUBLIC_SUPABASE_URL: https://example.supabase.co
+          NEXT_PUBLIC_SUPABASE_ANON_KEY: dummy-anon-key-for-build
+
+      - name: Build result comment
+        id: comment-body
+        if: always() && steps.lhci.outputs.manifest != ''
```

actionlint (1.7.12)

[error] property "manifest" is not defined in object type {assertionresults: string; links: string; resultspath: string}

(expression)